styles/themes/light.scss

```diff
@@ -196,12 +206,39 @@ html.light-mode {
 	}
 
 	.wallet .buttons button {
-		background-color: rgba(243, 243, 243, 0.90);
+		background-color: rgba(243, 243, 243, 0.9);
 	}
 
 	.gallery .has-text-overflow-ellipsis,
 	.collections .columns .card .card-content .collection-title,
-	.collections .columns .card .card-content .heading--inline {
+	.collections .columns .card .card-content .heading--inline,
+	.control-label {
 		color: $text-light-theme;
 	}
-}
+
+	.collection-card .card-content {
+		background-color: $frosted-glass-background-dark  !important;
+	}
+
+	.price-block__original {
+		.money {
+			color: $text-light-theme;
+		}
+	}
+
+	.share__tooltip {
+		.tooltip-content {
+			background-color: $white  !important;
+		}
+	}
+
+	.carousel-items {
+		.title {
+			color: $text-light-theme;
+		}
+	}
+
+	.series-sticky-header th {
+		background: $white;
+	}
```

Comment:
```suggestion
```

works without

Reply:
@prachi00 let's address @roiLeo's comments and move on :)